docs/conf.py

```diff
@@ -106,7 +106,7 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = 'default'
+html_theme = 'classic'
```

Comment:
Nope, `classic` is the new name for `default`, and ReadTheDocs overrides it anyway - there should be no visible change